test/events/podtatohead.deploy-v0.1.2.triggered.json

```diff
@@ -0,0 +1,24 @@
+{
+  "data": {
+    "configurationChange": {
+      "values": {
+        "image": "ghcr.io/j-poecher/podtato-head/podtatoserver:v0.1.2"
```

Comment:
Do we need the image from j-poecher, or can we use the official one, e.g.:
```
IMAGE="ghcr.io/podtato-head/podtatoserver"
VERSION=v0.1.1
SLOW_VERSION=v0.1.2
```

Reply:
It seems to work with the official image, I replaced it